packages/terra-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -258,7 +266,10 @@ const ColumnHeaderCell = (props) => {
       onKeyDown={(isSelectable || isResizable) ? handleKeyDown : null}
       style={{ width: `${width}px`, height: headerHeight, left: cellLeftEdge }} // eslint-disable-line react/forbid-dom-props
     >
-      <div className={cx('header-container')} role={displayName && 'button'}>
+      <div
+        className={cx('header-container')}
+        {...isSelectable && displayName && { ref: columnHeaderCellButtonRef, tabIndex: '-1', role: 'button' }}
```

Comment:
This won't work properly for the Terra Table.  In that scenario, the tabIndex would be 0.  Also, should the cell tabIndex be updated when this one is set?  Would the cell ever have focus in that use case?

Reply:
We might be able to get away with it since the button is taking up most of the area.  Ideally, the cell itself would not get focus when there is a button.